src/v2/Apps/Auctions/Components/MyBids.tsx

```diff
@@ -0,0 +1,207 @@
+            },
+          } = lot
+
+          const reserveNotMet =
+            reserveStatus === "ReserveNotMet" && !isLiveAuction
+
+          return (
+            <RouterLink to={href} noUnderline>
+              <Flex alignItems="center">
+                <Box pr={0.5}>
+                  <Image src={image.cropped.url} />
+                </Box>
+                <Box width="100%">
+                  <Flex justifyContent="space-between" alignItems="center">
+                    <Box>
+                      <Text variant="caption">Lot {lotLabel}</Text>
+                      <Text variant="caption">{artistNames}</Text>
+                      <Flex>
+                        <Text variant="caption" mr={0.5}>
+                          {sellingPrice.display}
+                        </Text>
+                        <Text variant="caption" color="black60">
+                          ({bidCount} {bidCount === 1 ? "bid" : "bids"})
+                        </Text>
+                      </Flex>
+                    </Box>
+                    <Box textAlign="right">
+                      <Box>
+                        <Flex>
+                          {reserveNotMet ? (
+                            <ReserveNotMet />
+                          ) : lot.isHighestBidder ? (
+                            <HighestBid />
+                          ) : (
+                            <Outbid />
+                          )}
+                        </Flex>
+                        <Spacer my={0.5} />
+                        <Button variant="secondaryOutline" size="small">
+                          Bid
+                        </Button>
+                      </Box>
+                    </Box>
+                  </Flex>
+                </Box>
+              </Flex>
+            </RouterLink>
+          )
+        })}
+      </Join>
+    </Box>
+  )
+}
+
+export const MyBidsFragmentContainer = createFragmentContainer(MyBids, {
+  me: graphql`
+    fragment MyBids_me on Me {
+      name
+      auctionsLotStandingConnection(first: 25) {
+        edges {
+          node {
+            isHighestBidder
+            lot {
+              saleId
+              soldStatus
+              internalID
+              bidCount
+              reserveStatus
+              soldStatus
+              askingPrice: onlineAskingPrice {
+                display
+              }
+              sellingPrice: floorSellingPrice {
+                display
+              }
+            }
+            saleArtwork {
+              position
+              lotLabel
+              artwork {
+                artistNames
+                href
+                image {
+                  cropped(width: 70, height: 70) {
+                    url
+                  }
+                }
+              }
+              sale {
+                internalID
+                liveStartAt
+                endAt
+                status
+              }
+            }
+          }
+        }
```

Comment:
I copied this query stuff from Eigen, need to clean up unused props in a follow-up